app/src/main/java/org/ole/planet/myplanet/utils/SyncTimeLogger.kt

```
@@ -81,22 +89,37 @@ object SyncTimeLogger {
+                "http://$urlString"
+            } else {
+                urlString
+            }
+            val url = URL(formattedUrl)
+            val conn = url.openConnection() as HttpURLConnection
+            conn.requestMethod = "GET"
+            conn.connectTimeout = 5000
+            conn.readTimeout = 5000
+            conn.connect()
+            conn.responseCode in 200..299
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)